bemuse/package.json

```diff
@@ -28,88 +28,102 @@
   "author": "Thai Pangsakulyanont <dtinth@spacet.me> (http://dt.in.th/)",
   "license": "AGPL-1.0",
   "devDependencies": {
+    "@ephesoft/webpack.istanbul.loader": "^2.2.0",
```

Comment:
This dependency concerns me a bit because it is less popular than https://www.npmjs.com/package/@jsdevtools/coverage-istanbul-loader

I'm curious if you have tried that dependency already?

Reply:
It seems to work well but not maintained for a while. [However I found this package by sorting by maintainance](https://www.npmjs.com/search?ranking=maintenance&q=istanbul%20loader) for security.

Reply:
[On first glace Ephesoft seems to be a reputable company](https://ephesoft.com/), so yeah, let’s go with this package. 👍